content/02.introduction.md

```diff
@@ -0,0 +1,12 @@
+## Introduction
+
+1. Talk about knowledge graphs
+  1. Define what they are
+  2. Provide a diagram to orient the reader
+2. Mention how these graphs are constructed 
+3. Mention some applications these graphs can be used for
+	1. Biomedical setting
+	2. Social Media 
+	3. Question and answering systems 
+	4. more if it comes to mind
+4. Conclude section by mention that this review dives deeper into approaches used for construction and applying these graphs
```

Comment:
Might be good to note towards the conclusion once more that we're focusing on biomedical applications